src/utils.js

```diff
@@ -0,0 +1,93 @@
+// user selects one of these options when first adding an item to a list
+export const radioButtonOptions = [
+  {
+    option: 'soon',
+    value: 7,
+    defaultChecked: true,
+  },
+  {
+    option: 'kind of soon',
+    value: 14,
+    defaultChecked: false,
+  },
+  {
+    option: 'not soon',
+    value: 30,
+    defaultChecked: false,
+  },
+];
+
+//used to normalize item names when added by a user
+export const removePunctuation = (string) => {
+  const punctuationlessString = string
+    .toLowerCase()
+    .replace(/[^\w\s]|_/g, '')
+    .split(' ')
+    .join('');
+
+  return punctuationlessString;
+};
+
+// general variables used for time calculations
```

Comment:
you might have enough strings vs time utilities to have two separate files (utilitiesString and utilitiesTime), but this is ok too